packages/blocks/microsoft-teams/src/lib/common/chat-types.ts

```diff
@@ -0,0 +1,12 @@
+export enum ChatTypes {
```

Comment:
I think it's confusing that we have CHAT_TYPE and ChatTypes.

ChatTypes -- are there other options? since CHAT_TYPE has 4 and this is 2.